packages/strapi-plugin-content-type-builder/admin/src/containers/HomePage/index.js

```diff
@@ -54,6 +70,20 @@ export class HomePage extends React.Component { // eslint-disable-line react/pre
     this.props.deleteContentType(contentTypeName, this.context);
   }
 
+  handleShortCut = (e) => {
+    if (includes(keyBoardShortCuts, e.keyCode)) {
+      const mapKeys = clone(this.state.mapKeys);
+      mapKeys[e.keyCode] = e.type === 'keydown';
+      this.setState({ mapKeys });
+
+      // Check if user pressed option + n;
+      if (mapKeys[18] && mapKeys[78]) {
```

Comment:
`18` and  `78` values are already hardcoded at the top of this file. Maybe it could be more dynamic...